src/Command/MailerCitizenProjectValidatedSummaryCommand.php

```diff
@@ -0,0 +1,155 @@
+          ->setDescription('Sending a summary email of validated citizen projects to adherents.');
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $container = $this->getContainer();
+
+        $this->entityManager = $container->get('doctrine.orm.entity_manager');
+        $this->mailer = $container->get('app.mailer.transactional');
+        $this->logger = $container->get('logger');
+        $this->twig = $container->get('twig');
+
+        $this->adherentRepository = $this->entityManager->getRepository(Adherent::class);
+        $this->citizenProjectRepository = $this->entityManager->getRepository(CitizenProject::class);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $adherents = $this->adherentRepository->findAllEnabledAdherents();
+
+        $counter = 0;
+        foreach ($adherents as $chunk) {
+            /** @var Adherent $adherent */
+            $adherent = $chunk[0];
+
+            // Finds all citizen projects near the Adherent
+            $citizenProjects = $this->citizenProjectRepository->findNearByCitizenProjectSummariesForAdherent(
+                $adherent,
+                self::CITIZEN_PROJECTS_LIST_LIMIT,
+                $input->getOption('maxAge')
+            );
+
+            if (0 === count($citizenProjects)) {
+                continue;
+            }
+
+            // Broadcast the citizen projects and returns them formatted in array
+            $broadcastedCitizenProjects = $this->broadcast($citizenProjects);
+
+            // Sends the message
+            $message = $this->createMessage($broadcastedCitizenProjects, $adherent);
+            $this->mailer->sendMessage($message);
+
+            $this->logger->info(
+                sprintf(
+                    '[Broadcasting citizen projects] An email with %d CP has been queued for adherent #%d',
+                    count($citizenProjects),
+                    $adherent->getId()
+                )
+            );
+            ++$counter;
+        }
+        $this->logger->info(sprintf('[Broadcasting citizen projects] Total emails queued : %d', $counter));
+    }
+
+    /**
+     * Puts the citizen projects broadcast flag and returns the list.
+     *
+     * @param array $citizenProjects List of CitizenProject
+     *
+     * @return array List of citizen projects formatted for the template
+     */
+    private function broadcast(array $citizenProjects): array
+    {
+        $broadcastedCitizenProjects = [];
+
+        try {
+            $this->entityManager->beginTransaction();
+
+            /** @var CitizenProject $citizenProject */
+            foreach ($citizenProjects as $citizenProject) {
+                if ($citizenProject->isBroadcasted()) {
+                    continue;
+                }
+
+                $citizenProject->setBroadcasted(true);
+                $this->entityManager->persist($citizenProject);
```

Comment:
This line is useless, the entity is already tracked by doctrine since it has fetched it from the db.